FBC/DebtAndEquities/CreditEvents.rdf

```diff
@@ -185,11 +191,11 @@
 	<owl:Class rdf:about="&fibo-fbc-dae-cre;ObligationRestructuring">
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-cre;ObligationSpecificCreditEvent"/>
 		<rdfs:label xml:lang="en">obligation restructuring</rdfs:label>
-		<skos:definition xml:lang="en">entity-specific credit event that materially impacts an obligation, such as an interest rate reduction, principal reduction, deferral of interest or principal, change in priority ranking, or change in currency or composition of payment</skos:definition>
+		<skos:definition xml:lang="en">credit event that materially impacts an obligation, such as an interest rate reduction, principal reduction, deferral of interest or principal, change in priority ranking, or change in currency or composition of payment</skos:definition>
```

Comment:
better IMO to separate the examples to the specific annotation property. Which makes clear the definition needs to say something about what "restructuring" means - specifically a change in terms to reduce the burden on the party, typically to avoid the risk of bankruptcy.

Reply:
Can we address this in a separate issue?  I don't think it is hugely important to do that in this pass.

Reply:
Raised #1157